platform/mac/AppDelegate.mm

```diff
@@ -1107,7 +1107,7 @@ - (void) loadExtensionMenu
     NSMenu *appMenu = [[NSApplication sharedApplication] mainMenu];
     NSMenuItem *windowMenuItem = [appMenu itemWithTitle:kWindowMenuItemName];
     NSMenu *windowMenu = [windowMenuItem submenu];
-    long welcomeItemIdx = [windowMenu indexOfItemWithTitle:@"Welcome to Corona"];
+    long welcomeItemIdx = [windowMenu indexOfItemWithTitle:@"Welcome to Solar2D"];
```

Comment:
This would cause an issue, since it is trying to fetch menu by the name. Will rename menu as well.

Reply:
Fixed welcome window title and menu item to be Solar2D